src/specs/api.yaml

```diff
@@ -561,4 +561,20 @@ paths:
                   experimentId:
                     type: string
                   samples:
-                    $ref: ./models/api-body-schemas/Samples.v1.yaml
\ No newline at end of file
+                    $ref: ./models/api-body-schemas/Samples.v1.yaml
+  '/projects':
+    get:
+      summary: ''
+      operationId: getProjects
+      x-eov-operation-id: projects#get
```

Comment:
@aerlaut isn't this out of scope from Stefan's ticket (https://biomage.atlassian.net/browse/BIOMAGE-616) and actually part of this ticket: https://biomage.atlassian.net/browse/BIOMAGE-953

Reply:
My ticket is for loading samples and projects, that endpoint needed to be created so i can do it

Reply:
It was one ticket and then split into two parts, the UI and API. The UI part requires the API endpoint. Stefan beats Martin in opening up the PR and implementing the endpoint, so :S